playbooks/cluster_setup_network.yaml

```diff
@@ -50,14 +50,14 @@
 
 - name: Configure Network
   become: true
-  hosts: cluster_machines
+  hosts: all
```

Comment:
I preferred using cluster_machines and standalone_machine because there is also the VMs.